### PR TITLE
Default to url

### PR DESCRIPTION
**Pull request by squat**
Add a default URL value for the `--to` flag of the telemeter client.

cc @s-urbaniak

### CODE REVIEW

File: cmd/telemeter-client/main.go [hunk 1]
```diff
@@ -192,7 +192,7 @@ func (o *Options) Run() error {
 	if len(o.ToUpload) > 0 {
 		to, err = url.Parse(o.ToUpload)
 		if err != nil {
-			return fmt.Errorf("--to is not a valid URL: %v", err)
+			return fmt.Errorf("--to-upload is not a valid URL: %v", err)
```
Comment (s-urbaniak) [inline cmd/telemeter-client/main.go]: the cli flag is still just `--to`, no?

Comment (s-urbaniak) [reply cmd/telemeter-client/main.go]: ah, that makes sense, thanks for fixing! :+1:

Comment (squat) [reply cmd/telemeter-client/main.go]: :+1:

**Pull request merged**